README.md

```diff
@@ -68,7 +68,7 @@ Prefix for the value of the cookie. This is useful for compatibility with `expre
 ##### cookie
 The options object is used to generate the `Set-Cookie` header of the session cookie. May have the following properties:
 * `path` - The `Path` attribute. Defaults to `/` (the root path).
-* `maxAge` - A `number` in milliseconds that specifies the `Expires` attribute by adding the specified milliseconds to the current date. If both `expires` and `maxAge` are set, then `maxAge` is used.
+* `maxAge` - A `number` in seconds that specifies the `Expires` attribute by adding the specified seconds to the current date. If both `expires` and `maxAge` are set, then `maxAge` is used.
```

Comment:
Can you point out where it is being interpreted as seconds? I understand the cookie spec defines it as seconds, but the implementation looks to be in milliseconds:

https://github.com/fastify/session/blob/25b4f6ba4814385a31293b13ddc5f57cac0cf8e2/lib/cookie.js#L42-L46

Reply:
What are use cases setting `maxAge` in milliseconds? Seems like it's a common practice to set in seconds

Reply:
Yeah, I think we should swap out this cookie module with the one from fastify/cookie

Reply:
But that would means that major version should be increased, which doesn't seem like a significant change) So probably a fix to the typings would be more then enough

Reply:
I see - can you point to where the issue is in types?

Reply:
> What are use cases setting `maxAge` in milliseconds?

JavaScript's date handling is based upon milliseconds, not seconds.